_guide/_pages/security/aws.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Amazon Web Services (AWS) 
+sidenav: security
+sticky_sidenav: true
+---
+A common practice is storing files in [Amazon Simple Storage Service](https://docs.aws.amazon.com/s3/index.html) (Amazon S3).
+S3 gives the developer an easy way to store data in the cloud in a "bucket", and download it when needed. But this 
+introduces security concerns.
+
+When thinking about how to upload or download data in your application, there are always tradeoffs to think about -- often
+processes that are easier to use are less secure; likewise a workflow that is more complex is often more secure. 
+
+These tradeoffs get more significant depending on the FISMA level of your system.
+
+### Presigned URLs 
+All objects in S3 are private by default. This is great for security practices, but can make it tricky if you want to allow users to upload or download objects programmatically via an application. A method of allowing users to transfer
+data without AWS credentials is to use [presigned urls](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html).
+
+#### Things to know about presigned URLs
```

Comment:
[This blog post](https://advancedweb.hu/are-s3-signed-urls-secure/) goes into even more detail — thoughts on linking there, and keeping this page focused on recommendations specific to our (government) context?

Reply:
that's a good idea, although like above, want to give folks enough context so they don't have to click away to get the general point.

what do you think of keeping the top bullet list (perhaps slightly more fleshed out) and then a link to the blog post in case folks want more details?

Reply:
i'm also a little nervous about linking to external tech articles -- both bc that can feel like an endorsement and also because they can change/move. (our documentation at TTS is somewhat riddled with broken links.) do you have any thoughts about that?

Reply:
My only comment on this is that the "Misconfigured caching" section of the linked article is interesting and might be worth citing/mentioning in some form. That's a "way things could go wrong" that might not be obvious